resources/quality/volumic/stream30ultrasc_coarse.inst.cfg

```diff
@@ -0,0 +1,30 @@
+[general]
+version = 4
+name = Bas
+definition = stream30ultrasc
+
+[metadata]
+setting_version = 16
+type = quality
+quality_type = coarse
+weight = -1
+global_quality = True
+
+[values]
+layer_height = 0.250
+infill_sparse_density = 25
+top_bottom_thickness = 1
+wall_thickness = 1.2
+fill_outline_gaps = True
```

Comment:
Another setting that always has the same value; it's best to define this in the machine profile

Reply:
all parameters have been put in global profil except two of them that change. Done in every profils.